packages/predict/build.sh

```diff
@@ -0,0 +1,9 @@
+
+TERMUX_PKG_HOMEPAGE=https://www.qsl.net/kd2bd/predict.html
+TERMUX_PKG_DESCRIPTION="Satellite tracking, orbital prediction."
+TERMUX_PKG_LICENSE="GPL-3.0"
```

Comment:
In the tarball, `LICENSE` shows GPL-3.0 but `COPYING` shows GPL-2.0.

The head of `predict.c` indicates that the license is GPL-2.0-or-later. So I suggest that `TERMUX_PKG_LICENSE` be

```bash
TERMUX_PKG_LICENSE="GPL-2.0, GPL-3.0"
```